examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,76 @@
+"""
+Multi-subject connectome for ADHD
+=================================
+
+This example estimates a connectome from spheres in the brain. The coordinates
+of the spheres have been taken from a meta-analysis on neurosynth website.
+
+"""
+import matplotlib.pyplot as plt
+
+
+# Fetching datasets ###########################################################
+print("-- Fetching datasets ...")
+from nilearn import datasets
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+
+# print basic information on the dataset
+print('First subject functional nifti image (4D) is at: %s' %
+      adhd_dataset.func[0])  # 4D data
+
+
+# Extracting region signals ###################################################
+from nilearn import image
+from nilearn import input_data
+
+from sklearn.externals.joblib import Memory
+mem = Memory('nilearn_cache')
+
+print("... Extracting time series ...")
```

Comment:
> I don't know why but verbose is activated for maskers in all examples.

Let's change that for this example. It makes sense for examples with many
subjects, but with only one subject it does not make sense.